src/stories/Forms/Select/Option.tsx

```diff
@@ -7,7 +7,8 @@ const Wrapper = styled.li`
   display: flex;
   position: relative;
   align-items: center;
-  padding: ${rem(13)} var(--spacing-16);
+  height: var(--spacing-48);
+  padding: 0 var(--spacing-16);
```

Comment:
As discussed yesterday, I think we should continue to use margins here to avoid breaking UI when using long option labels.